LayoutTests/fonts/font-weight-invalid-crash.html

```diff
@@ -0,0 +1,32 @@
+<style>
+#x30,*:empty { normal;-webkit-transition-property: all;border-right-color: }
+*:root,.class0 { separate;font-stretch: expanded;-webkit-padding-before: 'a b';-webkit-column-break-inside: farthest-side invert auto;font-weight: 1;inset-inline-end: }
+progress,*:scope,.class3 { 0px;animation: keyframes4,keyframes1 cubic-bezier(0,358,1,1) }
+#x50:placeholder-shown,#x73:dir(ltr),*:defined { space;-webkit-animation-duration: 1s;border-bottom-style: auto;transition-duration: 1.5s;rotation: 180deg;-webkit-transition-timing-function: step-start,cubic-bezier(1,0,0,16);right: }
+*::before { -webkit-text-orientation: upright;-webkit-transition-property: }
```

Comment:
We shouldn’t need all of this stuff just to test this one change. I recommend creating a new test from scratch.

Reply:
I've reduced the test as much possible, and further reducing it doesn't cause the issue to reproduce.